PocketKit/Sources/PocketKit/Resources/en.lproj/Localizable.strings

```diff
@@ -156,6 +156,10 @@
 "settings.accountManagement.deleteAccount.deletionConfirmation" = "You understand your Pocket account and data will be **permanently deleted.**";
 //Links to a help article on how to cancel your subscription
 "settings.accountManagement.deleteAccount.howToCancel" = "How to cancel your Pocket Premium subscription";
+//Loading screen showing that the account deletion is in progress
+"settings.accountManagement.deleteAccount.deleting" = "Deleting your account";
+"settings.accountManagement.deleteAccount.error.headline" = "Oops! Try again?";
```

Comment:
for strings that already exist like `Oops! Try again?` should we make it more general so that we don't translate it twice?